2019-12-03 15:01:17 =CRASH REPORT====
  crasher:
    initial call: Elixir.GenAMQP.Conn:init/1
    pid: <0.595.0>
    registered_name: []
    exception error: {{badmatch,{error,econnrefused}},[{'Elixir.GenAMQP.Conn',init,1,[{file,"lib/gen_amqp/conn.ex"},{line,51}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,374}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,342}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
    ancestors: ['Elixir.StaticConnSup','Elixir.GenAMQP.AppSupervisor',<0.592.0>]
    message_queue_len: 0
    messages: []
    links: [<0.594.0>]
    dictionary: [{gen_server_call_timeout,60000}]
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 27
    reductions: 28998
  neighbours:
2019-12-03 15:01:17 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.StaticConnSup'}
     Context:    start_error
     Reason:     {{badmatch,{error,econnrefused}},[{'Elixir.GenAMQP.Conn',init,1,[{file,"lib/gen_amqp/conn.ex"},{line,51}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,374}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,342}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
     Offender:   [{pid,undefined},{id,'Elixir.ConnHub'},{mfargs,{'Elixir.GenAMQP.Conn',start_link,[<<"amqp://guest:guest@localhost">>,'Elixir.ConnHub']}},{restart_type,transient},{shutdown,5000},{child_type,worker}]

2019-12-03 15:01:17 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.GenAMQP.AppSupervisor'}
     Context:    start_error
     Reason:     {shutdown,{failed_to_start_child,'Elixir.ConnHub',{{badmatch,{error,econnrefused}},[{'Elixir.GenAMQP.Conn',init,1,[{file,"lib/gen_amqp/conn.ex"},{line,51}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,374}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,342}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}}}
     Offender:   [{pid,undefined},{id,'Elixir.StaticConnSup'},{mfargs,{'Elixir.GenAMQP.ConnSupervisor',start_link,['Elixir.StaticConnSup',[{'Elixir.ConnHub',<<"amqp://guest:guest@localhost">>}]]}},{restart_type,permanent},{shutdown,infinity},{child_type,supervisor}]

2019-12-03 15:01:17 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.591.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,'Elixir.StaticConnSup',{shutdown,{failed_to_start_child,'Elixir.ConnHub',{{badmatch,{error,econnrefused}},[{'Elixir.GenAMQP.Conn',init,1,[{file,"lib/gen_amqp/conn.ex"},{line,51}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,374}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,342}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}}}}},{'Elixir.GenAMQP',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,138}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
    ancestors: [<0.590.0>]
    message_queue_len: 0
    messages: []
    links: [<0.590.0>,<0.592.0>,<0.42.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 27
    reductions: 217
  neighbours:
2019-12-03 15:09:50 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.335.0>
    registered_name: []
    exception exit: {{{already_started,<0.316.0>},{'Elixir.TestApp',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,138}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
    ancestors: [<0.334.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.336.0>,normal}]
    links: [<0.334.0>,<0.42.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 27
    reductions: 209
  neighbours:
2019-12-03 15:10:23 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.269.0>
    registered_name: []
    exception exit: {{{already_started,<0.250.0>},{'Elixir.TestApp',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,138}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
    ancestors: [<0.268.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.270.0>,normal}]
    links: [<0.268.0>,<0.42.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 27
    reductions: 209
  neighbours:
2019-12-03 15:10:52 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.278.0>
    registered_name: []
    exception exit: {{{already_started,<0.259.0>},{'Elixir.TestApp',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,138}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
    ancestors: [<0.277.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.279.0>,normal}]
    links: [<0.277.0>,<0.42.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 27
    reductions: 209
  neighbours:
2019-12-03 15:13:43 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.278.0>
    registered_name: []
    exception exit: {{{already_started,<0.259.0>},{'Elixir.TestApp',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,138}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
    ancestors: [<0.277.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.279.0>,normal}]
    links: [<0.277.0>,<0.42.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 27
    reductions: 217
  neighbours:
2019-12-03 15:15:29 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.290.0>
    registered_name: []
    exception exit: {{{already_started,<0.271.0>},{'Elixir.TestApp',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,138}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
    ancestors: [<0.289.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.291.0>,normal}]
    links: [<0.289.0>,<0.42.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 27
    reductions: 209
  neighbours:
2019-12-03 16:08:14 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.395.0>
    registered_name: []
    exception exit: {{{already_started,<0.376.0>},{'Elixir.TestApp',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,138}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
    ancestors: [<0.394.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.396.0>,normal}]
    links: [<0.394.0>,<0.42.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 27
    reductions: 209
  neighbours:
2019-12-03 16:08:40 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.650.0>
    registered_name: []
    exception exit: {{{already_started,<0.631.0>},{'Elixir.TestApp',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,138}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
    ancestors: [<0.649.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.651.0>,normal}]
    links: [<0.649.0>,<0.42.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 27
    reductions: 209
  neighbours:
2019-12-03 16:11:29 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.650.0>
    registered_name: []
    exception exit: {{{already_started,<0.631.0>},{'Elixir.TestApp',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,138}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
    ancestors: [<0.649.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.651.0>,normal}]
    links: [<0.649.0>,<0.42.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 27
    reductions: 209
  neighbours:
2019-12-03 16:17:08 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.AppSupervisor'}
     Context:    start_error
     Reason:     {already_started,<0.268.0>}
     Offender:   [{pid,undefined},{id,'Elixir.StaticConnSup'},{mfargs,{'Elixir.GenAMQP.ConnSupervisor',start_link,['Elixir.StaticConnSup',[{'Elixir.ConnHub',<<"amqp://guest:guest@localhost">>}]]}},{restart_type,permanent},{shutdown,infinity},{child_type,supervisor}]

2019-12-03 16:17:08 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.286.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,'Elixir.StaticConnSup',{already_started,<0.268.0>}}},{'Elixir.TestApp',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,138}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
    ancestors: [<0.285.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.287.0>,normal}]
    links: [<0.285.0>,<0.42.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 27
    reductions: 210
  neighbours:
2019-12-03 16:20:47 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.286.0>
    registered_name: []
    exception exit: {{bad_return,{{'Elixir.TestApp',start,[normal,[]]},{'EXIT',{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"expected :strategy option to be given">>},[{'Elixir.Supervisor',init,2,[{file,"lib/supervisor.ex"},{line,609}]},{'Elixir.Supervisor',start_link,2,[{file,"lib/supervisor.ex"},{line,564}]},{application_master,start_it_old,4,[{file,"application_master.erl"},{line,277}]}]}}}},[{application_master,init,4,[{file,"application_master.erl"},{line,138}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
    ancestors: [<0.285.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.287.0>,normal}]
    links: [<0.285.0>,<0.42.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 27
    reductions: 229
  neighbours:
2019-12-03 16:39:45 =ERROR REPORT====
** Generic server 'Elixir.TestServers.NothingServer_meck' terminating 
** Last message in was {'EXIT',<0.327.0>,killed}
** When Server state == {state,'Elixir.TestServers.NothingServer',[{'__info__',1},{execute,2},{init,1},{noreply,0},{reply,1},{start_link,0},{child_spec,1}],{dict,1,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[],[],[],[],[[{execute,2}|{{execute,2},[{{args_matcher,['_','_'],#Ref<0.256133628.3021864962.122557>,false},{meck_exec,#Fun<Elixir.MicroUtils.AMQP.ServerTest.1.48467543>}}]}]],[],[],[],[],[],[]}}},true,[{<0.300.0>,{'Elixir.TestServers.NothingServer',execute,[[<<"message">>,#{event => <<"test_queue">>}],#{event => <<"test_queue">>}]},noreply}],{false,no_passthrough_cover},false,false,false,undefined,[]}
** Reason for termination == 
** killed
2019-12-03 16:39:45 =CRASH REPORT====
  crasher:
    initial call: meck_proc:init/1
    pid: <0.329.0>
    registered_name: 'Elixir.TestServers.NothingServer_meck'
    exception exit: {killed,[{gen_server,decode_msg,9,[{file,"gen_server.erl"},{line,432}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
    ancestors: [<0.327.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 27
    reductions: 2431856
  neighbours:
